src/System.Runtime.Environment/src/OSName.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.InteropServices
+{
+    public struct OSName : IEquatable<OSName>
```

Comment:
The [design guidelines](https://msdn.microsoft.com/en-us/library/141e06ef(v=vs.71).aspx) say to use all uppercase when the acronym is two letters, so this is correct.